internal/resolution/pm/maven/job_test.go

```diff
@@ -18,46 +18,60 @@ func TestNewJob(t *testing.T) {
 
 func TestRunCmdErr(t *testing.T) {
 	cases := []struct {
+		name  string
 		error string
 		doc   string
 	}{
 		{
+			name:  "General error",
 			error: "cmd-error",
 			doc:   util.UnknownError,
 		},
+
+		{
+			name:  "Mvn not found",
+			error: "        |exec: \"mvn\": executable file not found in $PATH",
+			doc:   "Mvn wasn't found. Please check if it is installed and accessible by the CLI.",
```

Comment:
Not sure but maybe we should call it Maven instead of Mvn as a name (despite the executable being named mvn). But It is not really important.